0_intro/exercises/bubble_sort/src/lib.rs

```diff
@@ -1,5 +1,20 @@
 pub fn bubble_sort(v: &mut Vec<i32>) {
-    todo!()
+    if v.len() == 0 {
+        return;
+    }
+
+    let mut ok: bool = false;
```

Comment:
```suggestion
    let mut ok = false;
```
No need to add the type, there is type inference in Rust. But it's not bad at all :) Sometimes you'll have to specify it tho